orchester_cms_integration/service.py

```diff
@@ -46,17 +46,23 @@ def get_user_status_list(member):
     username_getter = MEMBER_CONNECTOR_FIELD_GETTERS.get(connector_type)
 
     username = default_username
+    service_error = None
     try:
       is_active = do_action('check', connector_type, member)
       username = username_getter(member)
-    except:
+    except Exception as error:
+      error_template = "Failed to check if the user({}) is valid for {}"
```

Comment:
how about f strings ?

`f"Failed to check if the user({member.id}) is valid for {connector_type}"`

Reply:
💃

Reply:
TIL